backend/src/main/java/com/example/backend/service/EquipmentItemService.java

```diff
@@ -19,4 +20,13 @@ public EquipmentItemService(EquipmentItemRepository equipmentItemRepository) {
     public List<EquipmentItem> getEquipmentItems() {
         return equipmentItemRepository.findAll();
     }
+
+    public EquipmentItem addEquipmentItem(EquipmentItemDTO equipmentItemDTO) {
+        return equipmentItemRepository.save(EquipmentItem.builder()
```

Comment:
eventuell das .save zu .insert abändern
https://stackoverflow.com/questions/16209681/what-is-the-difference-between-save-and-insert-in-mongo-db